service/local-dev/setup_local_env.sh

```diff
@@ -33,6 +33,17 @@ rm -rf terra-helm/.git
 git clone -b "$TERRA_HELMFILE_BRANCH" --single-branch --depth=1 ${helmfilegit}
 rm -rf terra-helmfile/.git
 
+# Render manifests to terra-helmfile/output/ directory.
+#
+# Unfortunately we need to render them all into a single mega-file
+# because Skaffold's `kubectl` deployment does not support
+# recursive globbing like "output/manifests/**/*.yaml"
+mkdir -p ./terra-helmfile/output
+./terra-helmfile/bin/render \
+  -e "${ENV}" \
```

Comment:
Does this $ENV match the helmfile environments?
For my deployment, I would enter `dd` into this script.

Reply:
Yes, `-e` here refers to helmfile environment

Reply:
So `dd` is a helmfile environment?

Reply:
Yep! You can see it defined here: https://github.com/broadinstitute/terra-helmfile/blob/master/helmfile.yaml#L89

Reply:
Cool! Thank you